sys/net/gnrc/sock/include/gnrc_sock_internal.h

```diff
@@ -96,6 +97,25 @@ static inline bool gnrc_ep_addr_any(const sock_ip_ep_t *ep)
     return true;
 }
 
+/**
+ * @brief   Initializes a sock end-point from a given and sets the network
+ *          interface implicitly if there is only one potential interface.
+ * @internal
+ */
+static inline void gnrc_ep_set(sock_ip_ep_t *out, const sock_ip_ep_t *in,
+                               size_t in_size)
+{
+    memcpy(out, in, in_size);
```

Comment:
Is there a reason as not to fix the size to `sizeof(sock_ip_ep_t)` here (and remove the extra argument with it)?

Reply:
Yes, in https://github.com/RIOT-OS/RIOT/pull/8470/files#diff-71ddba2942574ad9a8d73c8de31b2230R126 this is called with `sizeof(sock_udp_ep_t)` ;-).

Reply:
Never mind then, It all looked like the same `sock_something_ep_t` from here :)